### ISSUE TITLE
Stories cannot be added to a group.

### ISSUE THREAD

**Issue by Malozi** [knighty/fimfiction-issues#305]
TL;DR at bottom. Hello, I'm an admin for the group known as Let's Make You Known! and we (maybe this affects us specifically) have a bit of an issue. So one of our folders was deleted at some point, which contained a multitude of other people's folders, (each folder belongs to a specific person, it is theirs to post their stories in) which I have since restored. Unfortunately, every time I or anyone else tries to add stories from that specific person back into their folder, all we see is "Server Error" and we can't add the story.

This has been going on since February 16th, and I don't know if this could possibly affect other groups or just ours, or how long this... bug/glitch/whatever has actually been going on.

TL;DR. Folders containing people's stories deleted. People's stories can no longer be added to group due to Server Error message. What do?
